webhooks: Update the docs for a few more integrations.

eeshangarg opened the pull request:
@rishig, @timabbott: These were a bit easier but took some time to thoroughly test them! Thanks! :)

eeshangarg commented:
@rishig, @timabbott: Updated! :)

rishig commented:
Awesome, thanks! Merged the first four, with a few changes.

A few comments on the other two:
* Capistrano: Do we still use humbug, e.g. in `task :humbug do`? Also it's a little confusing to call them python bindings, when we're calling it from ruby.
* Git: I'm a little confused by 
![image](https://user-images.githubusercontent.com/890911/39111575-71107030-468a-11e8-9524-781d6bc820d0.png)

Also, we create a `commits` stream, but it isn't referenced in any configuration on the git server?

timabbott commented:
For capistrano, I bet we do -- that integration hasn't been worked on since way back when.

eeshangarg commented:
@rishig: Thanks for reviewing! I just updated this.

* **Capistrano**: It isn't confusing to call them Python bindings. The Capistrano config file that uses the `zulip-send` utility is written in Ruby, but the `zulip-send` utility itself is written in Python and is a part of our Python bindings and is the main tool used to send the message here.

* **Git**: The path to the `integrations/` folder was wrong, I just fixed that. As for the "commits stream not being referenced in any configuration on the Git server", I think if you were to look at [`python-zulip-api/zulip/integrations/git/`](https://github.com/zulip/python-zulip-api/tree/master/zulip/integrations/git), you'll see that the `post-receive` script calls `commit_notice_destination` in `zulip_git_config.py`, which specifies `commits` as the target stream for notifications. Did I answer your question or am I missing something else here? :)

Let me know what you think! Thanks again!

\-Eeshan

rishig commented:
I see. In that case the following is not really correct; i.e. it has to be called commits, right?
![image](https://user-images.githubusercontent.com/890911/39142922-fa209ae8-46e0-11e8-86f2-8fed2d5a259c.png)

eeshangarg commented:
@rishig: You are absolutely correct, yes! I just fixed that. Sometimes in my haste to use macros wherever possible, I forget to take a final look at how everything comes together! I'll be more careful from now on, thanks for pointing this out! :)

eeshangarg commented:
@rishig: Updated! :)

eeshangarg commented:
@rishig: I also made a new PyPI release that includes the change that allows users to specify a different stream in `zulip_git_config.py`. :)

rishig commented:
Did you push the new version? This comment seems to still be there:
https://github.com/zulip/zulip/pull/9156#issuecomment-383656486

eeshangarg commented:
@rishig: I just updated/rebased this. Also, about the comment you mentioned, we don't need to change it now, I already made the stream name configurable [in this file here](https://github.com/zulip/python-zulip-api/blob/master/zulip/integrations/git/zulip_git_config.py#L26), so it actually is just a recommendation now! :) My apologies for responding so late to this, I got caught up with other projects and lost track of these PRs. Thanks! :)

eeshangarg commented:
@rishig: Updated :)

rishig commented:
For the codebase one the user still needs to make a bot, right?

eeshangarg commented:
@rishig: Updated! Sorry, I am so tired of reading the same thing again and again, I am starting to miss some of the most obvious stuff! :/

rishig commented:
yeah, I can imagine. How many are left?
(btw, merged!)
